src/plugins/plugin_integration/opensearch_dashboards.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": "pluginIntegration",
+  "version": "opensearchDashboards",
+  "server": true,
+  "ui": true,
+  "requiredPlugins": ["data", "visualizations", "savedObjects", "opensearchDashboardsUtils"]
```

Comment:
Why would it require visualizations plugin?

Reply:
Accidentally forgot to remove, thanks. It's not needed